client/gatsby-config.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path')
+const config = require('../utils/config.js')
+// require("dotenv").config({
+//   path: path.resolve(process.cwd(), '../env')
+// });
+const url = config.serverUrl
+
+module.exports = {
+  siteMetadata: {
+    title: 'Union for Street Solidarity',
+    author: 'USS',
+    description: 'TODO',
+  },
+  proxy: {
+    prefix: '/uploadedImages',
```

Comment:
This proxy thing isn't working. It would be easier to have template views imo. BBIAB